src/controller/java/src/chip/devicecontroller/ChipDeviceController.java

```diff
@@ -530,7 +531,31 @@ public void subscribeToEventPath(
         minInterval,
         maxInterval,
         false,
-        false);
+        false,
+        null);
+  }
+
+  public void subscribeToEventPath(
+      SubscriptionEstablishedCallback subscriptionEstablishedCallback,
+      ReportCallback reportCallback,
+      long devicePtr,
+      List<ChipEventPath> eventPaths,
+      int minInterval,
+      int maxInterval,
+      @Nullable Long eventMin) {
```

Comment:
Primitive data types cannot be null, so you use Long object instead of long, right?

Reply:
Yes.